pages/blog/graphql-deep-dive-5.mdx

```diff
@@ -0,0 +1,169 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: 'The Stack #3',
+  author: 'tvvignesh',
+  tags: ['graphql', 'deepdive', 'techstack'],
+  date: '2020-12-06',
+  description: 'The GraphQL Stack - Part 3',
+  image: '/blog-assets/graphql-deep-dive-5/cover.png',
+  thumbnail: '/blog-assets/graphql-deep-dive-5/cover.png',
+};
+
+export default withArticle({ ...meta });
+
+_This blog is a part of a series on GraphQL where we will dive deep into GraphQL and its ecosystem one piece at a time_
+
+- Part 1: [**Diving Deep**](/blog/graphql-deep-dive-1)
+- Part 2: [**The Usecase & Architecture**](/blog/graphql-deep-dive-2)
+- Part 3: [**The Stack #1**](/blog/graphql-deep-dive-3)
+- Part 4: [**The Stack #2**](/blog/graphql-deep-dive-4)
+- Part 5: [**The Stack #3**](/blog/graphql-deep-dive-5)
+
+In this series, we had looked at some of the interesting parts of the GraphQL stack so far with a range of tools, libraries and frameworks from the community.
+Let us continue the journey in this blog looking at more such tools and services which have created a great impact in the GraphQL ecosystem.
+
+**[GraphiQL](https://github.com/graphql/graphiql)**
+
+The evolution of GraphQL clients have been really amazing and I would say, this is one of the great things about GraphQL given its powerful introspection capabilities,
+being self documenting and also providing ability to extend everything with extensions.
+
+It all started with GraphiQL demonstrating all these back in the day, but then came Playground ([which had recently merged with the GraphiQL team](https://foundation.graphql.org/news/2020/04/03/web-based-graphql-ides-for-the-win-how-why-playground-graphiql-are-joining-forces/) to make things even more interesting),
+[Altair](https://github.com/imolorhe/altair) and even desktop/web/editor based clients like [Insomnia](https://insomnia.rest/), [Postman](https://www.postman.com/graphql/), [Hoppscotch](https://hoppscotch.io/), [VSCode Rest Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) and the list
+goes on all proving that the developer experience with GraphQL can be made really better with just some sugar on top.
```

Comment:
Really worth mentioning graphiql-explorer

Reply:
Thanks. Can't believe I left that out. Have given it a separate section though since it is more of a plugin to GraphiQL than a complete client. Pushed.